crates/sdk/src/env.rs

```diff
@@ -188,6 +188,14 @@ pub fn random_bytes(buf: &mut [u8]) {
     unsafe { sys::random_bytes(BufferMut::new(buf)) }
 }
 
+#[inline]
+pub fn send_proposal(value: &[u8], buf: &mut [u8]) {
+    unsafe { sys::send_proposal(Buffer::new(value), BufferMut::new(buf)) }
+}
+#[inline]
+pub fn approve_proposal(buf: &mut [u8]) {
+    unsafe { sys::approve_proposal(BufferMut::new(buf)) }
+}
```

Comment:
no need for these

Reply:
why?

Reply:
the `External` API is how developers will interface with these